internal/jstest/jstest.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jstest
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	// "github.com/FerretDB/dance/internal"
+)
+
+const uri = "mongodb://host.docker.internal:27017"
+
+func Run(ctx context.Context, args []string) error {
+
+	// bad
+	files := []string{}
+	for _, f := range args {
+		if strings.HasSuffix(f, ".js") {
+			wd, _ := os.Getwd()
+			ff := filepath.Join(wd, "mongo", f)
+			matches, err := filepath.Glob(ff)
+			if err != nil {
+				return err
+			}
+
+			for _, m := range matches {
+				i := strings.LastIndex(m, "/jstests")
+				m = m[i+1:]
+				files = append(files, m)
+			}
+		}
+	}
+
+	// builds the --eval statement
```

Comment:
What we are trying to do there?

Reply:
We are trying to load and execute the files in the shell using `eval` because providing a list of .js files with Docker compose does not seem to work. It will load the file and execute it.

```javascript
---
> load('jstests/core/all.js')
true
> 
```

Reply:
I think the problem is with shell globs, and it could be solved by Go code. For now, let's focus on one file without globs

Reply:
> I think the problem is with shell globs, and it could be solved by Go code. For now, let's focus on one file without globs

Yes that seems to be the problem. I left the logic in there for convenience and passing a single file doesn't break it.